epub33/rs/index.html

```diff
@@ -303,13 +303,13 @@ <h4>Parsing Relative URLs</h4>
 			<section id="sec-pkg-doc-base-dir">
 				<h4>Base Direction</h4>
 
-				<p id="confreq-rs-pkg-dir" data-tests="confreq-rs-pkg-dir.epub">If the <code>dir</code> attribute is set
+				<p id="confreq-rs-pkg-dir" data-tests="confreq-rs-pkg-dir.epub,confreq-rs-pkg-dir_rtl-001.epub,confreq-rs-pkg-dir_rtl-002.epub,confreq-rs-pkg-dir_rtl-004.epub,confreq-rs-pkg-dir_rtl-005.epub,confreq-rs-pkg-dir_rtl-006.epub">If the <code>dir</code> attribute is set
```

Comment:
Please remove my existing confreq-rs-pkg-dir.epub here; I added that presumptively a while back, but you are writing these tests instead so mine will not be committed.